GoogleForCreators/web-stories-wp
# Redesign: no selection "panel"

## Context



## Summary



## Relevant Technical Choices



## To-do



## User-facing changes
Adjusts the text typography in the no-selection panel.


## Testing Instructions
1. Open a story, unselect all elements
2. Verify the text in the panel looks as in [Figma](https://www.figma.com/file/bMhG3KyrJF8vIAODgmbeqT/Design-System?node-id=1799%3A48842).


### QA






This PR can be tested by following these steps:

1.

### UAT







This PR can be tested by following these steps:

1.
## Checklist




---



Fixes #6460
